roles/rpi/tasks/main.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: Disable and stop haveged
+  service: name=haveged.service enabled=no state=stopped
+  tags:
+    - rpi
```

Comment:
You tagged the whole with `rpi` in the playbook so the individual task tags here are redundant.